src/types/type-declarations.ts

```diff
@@ -111,3 +111,21 @@ export interface GoogleTagManagerApi {
   dataLayer: Record<string, any>[];
   google_tag_manager: GoogleTagManager;
 }
+
+/* Google Maps Embed */
+export interface GoogleMapsEmbedOptions {
```

Comment:
```suggestion
export interface GoogleMapsEmbedQueries {
```

Reply:
I think this should be suffixed with Queries instead of Options as `width` and `height` are missing.

`GoogleMapsEmbedOptions` should probably be something like 
`type GoogleMapsEmbedOptions = GoogleMapsEmbedQueries & GoogleMapsEmbedAttributes`

Reply:
ah I see - yeah, makes sense to differentiate between attributes and props... how about:
`GoogleMapsEmbedParams` ? in that way it matches "attributes" and "props" keys in the json files: https://github.com/GoogleChromeLabs/third-party-capital/blob/main/src/third-parties/google-maps-embed/data.json#L12